dockerfiles/ci/buster/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       context: .
```

Comment:
Once we agree on the result, I will remove `_tmp` and make the final build right before merging